### ISSUE TITLE
Add systemUsage to reduce Disk space required

### ISSUE THREAD

**Issue by cmoulliard** [gpe-mw-training/activemq-labs#11]
Add SystemUsage to Broker XML config

```
<systemUsage>
  <systemUsage>
    <memoryUsage>
      <memoryUsage limit="64 mb" />
    </memoryUsage>
    <storeUsage>
      <storeUsage limit="100mb" />
    </storeUsage>
    <tempUsage>
      <tempUsage limit="50mb" />
    </tempUsage>
  </systemUsage>
</systemUsage>
```

**Comment by cmoulliard** [comment]
Some students, on Windows machine, can't run the labs as the disk space doesn't support so big size (1Gb, ...)

**Comment by satyaj** [comment]
Fixed 57cdb63975d9a85df445ae25c4b21d6405424c02
